src/main/java/org/jenkinsci/plugins/docker/workflow/client/DockerClient.java

```diff
@@ -61,6 +63,12 @@
 
     private static final Logger LOGGER = Logger.getLogger(DockerClient.class.getName());
 
+    /**
+     * Maximum amount of time (in seconds) to wait for {@code docker} client operations which are supposed to be more or less instantaneous.
+     */
+    @Restricted(NoExternalUse.class)
+    public static int CLIENT_TIMEOUT = Integer.getInteger(DockerClient.class.getName() + ".CLIENT_TIMEOUT", 180); // TODO 2.4+ SystemProperties
```

Comment:
Still not a public API, but OK

Reply:
@oleg-nenashev Is it possible to overwrite this timeout in a script?

Reply:
Yes.

Reply:
is it somewhere documented howto do this?

Reply:
@jclick How is it possible to increase the timeout in a script? I still have the issue on "successful" builds being marked as "failed" because the docker rm fails with a 1.14 plugin.

Reply:
> Yes.

I am still having that issue at 180s timeout.
How do I increase that timeout to 240 or 300s?

Reply:
Via Java system property, but if you are asking the question, you likely have some bigger problem with your Docker daemon.

Reply:
@jglick via JAVA_OPTS environment variable?

Reply:
@jglick @oleg-nenashev could you provide instruction on how to custom the timeout?